code-env/python/spec/requirements.txt

```diff
@@ -1 +1 @@
-
+spacy==3.0
```

Comment:
Better to target a specific version (3.0.5 for instance)

Reply:
Can you add other necessary dependencies (cf. text preparation plugin)? The plugin can't work with only spacy.

Reply:
code-env/python/spec/requirements.txt

Reply:
That's a lot of dependencies compared to the Text preparation plugin. Where does it come from? Are you sure you need all of that?

Reply:
sorry, confusion with my code env, here : 8d52f5d